st2api/st2api/config.py

```diff
@@ -64,6 +64,7 @@ def _register_app_opts():
     CONF.register_opts(pecan_opts, group='api_pecan')
 
     logging_opts = [
+        cfg.BoolOpt('debug', default=False),
```

Comment:
There should already be existing system wide debug option (IIRC, `system.debug` which also gets set when `--debug` CLI flag is used).